dags/mantaray/mantaray.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""DAGs to run Mantaray benchmarks."""
+
+
+import datetime
+from airflow import models
+from xlml.utils import mantaray
+import yaml
+
+# Download xlml_jobs.yaml from GCS
+xlml_jobs_yaml = mantaray.load_file_from_gcs(
+    "gs://us-central1-ml-automation-s-bc954647-bucket/mantaray/xlml_jobs/xlml_jobs.yaml"
```

Comment:
should this bucket path be hard coded or configurable?

Reply:
I think we must use this path,  this path belongs to the storage bucket of the`ml-automation-solutions` composer environment. New jobs must be uploaded to this bucket in order to trigger DAG (re)creation.

Reply:
Ok that makes sense! Can you add a comment that clarifies the importance of this bucket?

Reply:
Thank you for the suggestion, added!